dbt/scripts/export_models.py

```diff
@@ -107,12 +107,8 @@ def parse_args():
         logger.info("Export completed successfully.")
 
     except Exception as e:
-        # The CloudWatch log handler does not include "ERROR - " at the
-        # beginning of log lines, so we need to preface the message with
-        # "ERROR - " to trigger alarms.
-        logger.error(f"ERROR - {e}")
-
-        # Suppress the error if we're uploading to Cloudwatch so that we can
-        # perform cleanup below, but otherwise, raise the error
-        if args.log_to_cloudwatch_group is None:
-            raise
+        logger.error(
+            str(e) or f"{type(e).__name__} raised with no message",
+            exc_info=True,
```

Comment:
Setting [`exc_info=True`](https://docs.python.org/3/library/logging.html#logging.Logger.debug) ensures that we push tracebacks to our logs, which will help for debugging errors in the future.